### DISCUSSION TITLE
Announcements center

### DISCUSSION THREAD

**Discussion by NaveedAhmadHematmal** [VDTS/CodeX.VaxineSolution#36] [Ideas]
We have to add the announcement's centre that organizations can share announcements of starting a new period, closing a period, holding a meeting and more with their employees.
Your ideas?

**Comment by SaeedaRasuly** [comment]
Yes, add this to the plan; I have no comments

**Reply by NaveedAhmadHematmal** [reply]
It should display only current announcements or it should work as a notification page and can see old ones as well?

**Reply by SaeedaRasuly** [reply]
In my point of view it should only display the latest annoucements I don't think there will be need of previous or old announcements

**Reply by YasinZahin** [reply]
I think it should work as a notification page. The history of announcements can help with the administration in future.

**Reply by NaveedAhmadHematmal** [reply]
@YasinZahin
But we are discussing employees app.

**Reply by NaveedAhmadHematmal** [reply]
Administration has already a history page.
